Sources/Confidence/Confidence.swift

```diff
@@ -147,6 +153,11 @@ extension Confidence {
             return self
         }
 
+        public func withVisitorId() -> Builder {
+            self.visitorId = VisitorUtil(storage: DefaultStorage.visitorIdCache()).getId()
```

Comment:
Did you consider to store this in `UserDefaults` instead?

Reply:
But! Makes sense to think about iCloud sync here as you commented in the kotlin... We don't want this data cloud synced between devices.

Reply:
Exactly, there is no obvious documentation on how to disable iCloud sync for `UserDefaults` that I can find.

I also see advantages in operating with a single storage solution that has obvious low level APIs and it doesn't come with any unexpected automatic behaviour from the OS, i.e. the files

Reply:
Pushed a commit with UserDefaults